workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -195,11 +190,24 @@ private static string[] ConstructArgs(string projectPath, string schemaCompilerP
             baseArgs.Add($"--native-output-dir=\"{toolsConfig.CodegenOutputDir}\"");
             baseArgs.Add($"--schema-path=\"{toolsConfig.SchemaStdLibDir}\"");
 
+            // Add user defined schema directories
             foreach (var schemaSourceDir in toolsConfig.SchemaSourceDirs)
             {
-                baseArgs.Add($"--schema-path=\"{schemaSourceDir}\"");
+                if (Directory.Exists(schemaSourceDir))
+                {
+                    baseArgs.Add($"--schema-path=\"{schemaSourceDir}\"");
+                }
+            }
+
+            // Add package schema directories
+            foreach (var directory in GetSchemaDirectories())
```

Comment:
Could rewrite here (and above) as something like 
`baseArgs.AddRange(GetSchemaDirectories().Select(dir => $"--schema-path=\"{directory}\""));`